docs/tutorial/rln-keystore-generator.md

```diff
@@ -0,0 +1,73 @@
+# rln-keystore-generator
+
+This document describes how to run and use the `rln-keystore-generator` tool. 
+It is meant to be used to generate and persist a set of valid RLN credentials to be used with rln-relay.
+
+## Pre-requisites
+
+1. An EOA with some ETH to pay for the registration transaction ($PRIVATE_KEY)
+2. An RPC endpoint to connect to an Ethereum node ($RPC_URL)
+
+## Usage
+
+1. First, we compile the binary
+    
+    ```bash
+    make -j12 rln-keystore-generator
+    ```
+    This command will fetch the rln static library and link it automatically.
+
+
+2. Define the arguments you wish to use
+
+    ```bash
+    export RPC_URL="https://sepolia.infura.io/v3/..."
```

Comment:
Should we provide `.env.example` with all these env vars (although some having example or empty values) rather than have a bunch of copies of these exports at various places - here, run-with-rln.md, onchain-rln-relay-chat2 has the values too (and could also reference the import/sourcing of `.env`)

It would potentially help us (you:) ) with keeping all the places referencing these in sync.

Reply:
agreed - follow up pr for this!